Mathlib/Order/RelSeries.lean

```diff
@@ -405,6 +414,9 @@ def snoc (p : RelSeries r) (newLast : α) (rel : r p.last newLast) : RelSeries r
 @[simp] lemma last_snoc (p : RelSeries r) (newLast : α) (rel : r p.last newLast) :
     (p.snoc newLast rel).last = newLast := last_append _ _ _
 
+@[simp] lemma last_snoc' (p : RelSeries r) (newLast : α) (rel : r p.last newLast) :
+    (p.snoc newLast rel).toFun (Fin.last (p.length + 1)) = newLast := last_append _ _ _
```

Comment:
```suggestion
    p.snoc newLast rel (Fin.last (p.length + 1)) = newLast := last_append _ _ _
```
or am I missing something?

Reply:
Inside some proofs about `CompositionSeries.Equivalent`, there are some `toFun` appeared somehow. This lemmas is meant to fix that.

Reply:
Aren't you rather missing a general lemma that `f.toFun = f`?

Reply:
I have added `toFun_eq_coe`. This lemmas seems still to be necessary, becasue the other lemma `last_def` only deals with `p.snoc newLast rel (Fin.last (p.snoc newLast rel).length)`

Reply:
Okay, I see what you mean. Can you add a comment that this lemma is only here for simp confluence?

Reply:
I have added a note